ideas/145-identity.md

```diff
@@ -0,0 +1,147 @@
+# Self-sovereign Identity
+
+## Preamble
+
+    Idea: 145-identity
+    Title: Self-sovereign Identity
+    Status: In Progress 
```

Comment:
Seems like idea as a whole is still in draft stage, but PoC is in progress? Not sure how we want to reflect this given all the requirements

Reply:
Yes, the "idea/swarm" is in draft, but the PoC is in progress. 
I'll sort out this.